docs/protocol.md

```diff
@@ -465,6 +465,7 @@ code   | name                 | description
 `0x05` | unexpected error     | Request resulted in an unexpected error. The request may have been completed before the error, retry only if the request is idempotent or duplicate execution can be handled.
 `0x06` | bad request          | Request args do not match expectations, request will never be satisfied, do not retry
 `0x07` | network error        | A network error (e.g. socket error) occurred.
+`0x08` | unhealthy            | Network refused to forward request to unhealthy node.
```

Comment:
I’ve adjusted the verbiage a little. Avoiding coupling the code to a specific reason for being declined since we may devise any number of systems and nodes along the network for autonomous health detection, all of which should have the same, "declined, don't retry" semantics.

That is, unless I’m wrong about the "don’t retry" semantics. If it’s appropriate to retry on an unhealthy request, the network will usually decline, but will allow a request through periodically to reassess the health.